plugins/liferay-maven-plugin/src/test/java/com/liferay/maven/plugins/ServiceBuilderVerifyTest.java

```diff
@@ -26,37 +26,147 @@
 
 /**
  * @author Gregory Amerson
+ * @author Simon Jiang
  */
 public class ServiceBuilderVerifyTest extends TestCase {
 
-	public void testBuildServiceResolveProject() throws Exception {
-		File testDir = ResourceExtractor.simpleExtractResources(
-			getClass(), "/projects/servicebuilder/6.2/resolveProject");
 
-		assertTrue(testDir.exists());
+    public void testServiceRresolveProject62() throws Exception {
```

Comment:
Lets not change the name of this test, since it was just about testing the resolve.